classes/OccurrenceIndividual.php

```diff
@@ -1190,14 +1272,14 @@ public function restoreRecord(){
 					$sql1 = 'INSERT INTO ommaterialsample(';
 					$sql2 = 'VALUES(';
 					foreach($secArr as $f => $v){
-						if(in_array(strtolower($f),$msFieldArr)){
-							$sql1 .= $f.',';
-							$sql2 .= '"'.$this->cleanInStr($v).'",';
+						if(in_array(strtolower($f), $msFieldArr)){
+							$sql1 .= $f . ',';
+							$sql2 .= '"' . $this->cleanInStr($v) . '",';
 						}
 					}
-					$sql = trim($sql1,' ,').') '.trim($sql2,' ,').')';
+					$sql = trim($sql1, ' ,') . ') ' . trim($sql2, ' ,') . ')';
```

Comment:
Actually, same with `trim()`. I wonder whether it will yell at us for not null checking.

Reply:
Nothing a little `?? ''` won't fix?